src/AspNetCoreModuleV2/RequestHandlerLib/requesthandler_config.h

```diff
@@ -209,6 +207,25 @@ class REQUESTHANDLER_CONFIG
         return &m_struConfigPath;
     }
 
+    VOID
+    ReferenceConfig()
```

Comment:
Why did you add manual Referencing/Dereferencing to the configuration?

Reply:
in the case of inprocess application LoadManagedApplication failed, we then create a failederrorapplication which shares config object with the previous application. As the config object will be owned by the application, we cannot call delete or use smart point, otherwise double free.

Reply:
failederrorapplication shouldn't share config with the previous application. Also if that's the case, can't you use a shared pointer?